CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.16.0-rc.0 / 2020-01-30
+
+* [CHANGE] PromQL: Remove MaxConcurrent from the PromQL engine opts #6712
+* [FEATURE] PromQL: Support trailing commas in grouping opts #6480
+* [FEATURE] PromQL: add absent_over_time query function #6490
+* [FEATURE] Adding optional logging of queries to their own file #6520
+* [ENHANCEMENT] Remote write: Improve sharding calculation in cases where we would always be consistently behind by tracking pendingSamples ##6511
```

Comment:
Bugfix?

Reply:
@csmarchbanks Thoughts? It's an enhancement to how the sharding calculation works imo, but it could just as easily been listed as a bugfix. Though I haven't seen anyone mention the same issue you and I were running into.

Reply:
I can see the argument either way, but I would say bugfix in this case. There were a couple other people reporting the issue in places like the Cortex channel.